docs/integrations/spark.rst

```diff
@@ -0,0 +1,122 @@
+Lance ❤️ Spark
+--------------------
+
+Lance can be used as a third party datasource of `<https://spark.apache.org/docs/latest/sql-data-sources.html>`_
+
+.. warning::
+    This feature is experimental and the APIs may change in the future.
+
+Build from source code
+~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
I wonder if this should come later, and the first section should be downloading prebuilt stuff. I think building from source wouldn't be that common, unless you are a developer. And the development docs might be something we want to be separate.